tabbycat/privateurls/templates/public_url_landing.html

```diff
@@ -57,7 +57,7 @@
       {% endif %}
     {% endif %}
 
-    {% if object.adjudicator and pref.participant_ballots == 'private-urls' %}
+    {% if object.adjudicator and pref.participant_ballots == 'private-urls' and debateadjudications|length > 0 %}
```

Comment:
Would it be possible to only show the current rounds for which they can submit ballots, rather than all current rounds if they're in at least one?